README.md

```diff
@@ -1,26 +1,32 @@
 # README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
 * Ruby version
    - 3.0.0p0 
 
-* System dependencies
-   - 
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+About the project: 
+Modern day science emphasizes the great importance of healthy and fulfilling interactions with food. Our quality of life and mental health is dependent upon on the daily food choices we make. Here at MegaBites we wish to empower you to make the choices that are right for you. We wanted to create an app where you can search recipes consistent with the top 4 diets backed by science and save any recipe you like to your own user recipe dashboard.   
+
+Built With:
+- React https://reactjs.org/docs/getting-started.html
+- Ruby on Rails https://guides.rubyonrails.org/
+- React Strap https://reactstrap.github.io/
+- React Router Dom https://reactrouter.com/web/guides/quick-start
+
+Getting Started: 
+Run these commands in terminal:
+1. git clone https://github.com/cheeseandhackers/Megabites.git
+2. cd Megabites
+3. bundle install
+4. yarn
+5. rails db:create
+6. rails db:migrate
+7. rails db:seed
+8. rails s
+9. open in preferred text editor
```

Comment:
it'd be also helpful to indicate which port the app gets served on (localhost:3000?) just so developers know what to expect quickly